python/src/SobolIndicesAlgorithmImplementation_doc.i.in

```diff
@@ -117,47 +113,65 @@ We are now going to mix these two samples to get an estimate of the sensitivity
    \right)
 
 For the sake of stability, computations will be performed with centered output.
-Let :math:`G_0` be the empirical mean of all collected output samples and define:
+Let :math:`\overline{\vect{g}}` be the sample mean of samples 
+:math:`\vect{g}(\mat{A})` and :math:`\vect{g}(\mat{B})`.
```

Comment:
```suggestion
Let :math:`\overline{\vect{g}}` be the mean of the combined samples 
:math:`\vect{g}(\mat{A})` and :math:`\vect{g}(\mat{B})`.
```

Reply:
Actually, both. It is the "sample mean of the combined samples". Fixed.